.develop/spec2.json

```diff
@@ -0,0 +1,5841 @@
+{
```

Comment:
Recommended that you use @open-rpc/examples here instead - this way you will not have to update your tests to know what version of OpenRPC your code works / doesnt work for.

Reply:
`spec2.json` is a file generated by the tests.

Well noted on the .@examples point, and will do when I start pulling the logic out and building   tests for it in isolation.